xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.h

```diff
@@ -17,6 +17,7 @@
 #include "threads/Event.h"
 #include "threads/SystemClock.h"
 #include "utils/Geometry.h"
+#include "utils/StreamDetails.h"
```

Comment:
Include not needed here.  Please remove and add the include to the cpp file instead.

Reply:
It was before used in header for variable definition, yes. Will move it.

Reply:
Let me resolve this comment once I've seen the change. ;-)